CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Changelog
 
 ### Improvements and Changes
 
--   Added some type hints to the `Program` class (@rht, gh-1115).
+-   Type hints have been added to the `Program` class (@rht, gh-1115, gh-1134)
```

Comment:
```suggestion
-   Type hints have been added to the `quil.py` file (@rht, gh-1115, gh-1134).
```

Reply:
you're doing the rest of the file right?

Reply:
Yes, the rest of the file.

Reply:
But there are 52 mypy errors, some are caused by another files that makes it impossible to reduce in this PR. How should I reword the commit message and the changelog message?

Reply:
Down to 37 errors.